src/languages/elixir.js

```diff
@@ -223,6 +223,10 @@ export default function(hljs) {
       })
     ]
   };
+  const OPERATOR = {
+    scope: 'operator',
```

Comment:
We also need `relevance: 0` to prevent this from breaking auto-detect since so few languages highlight operators yet.